src/physics/euler.h

```diff
@@ -277,15 +277,10 @@ class Euler : public PhysicsBase <dim, nstate, real>
     /** See the book I do like CFD, sec 4.14.2 */
     real compute_pressure_from_density_temperature ( const real density, const real temperature ) const;
 
-    /// The Euler split form is that of Kennedy & Gruber or Ismail & Roe.
+    /// The Euler split form convective flux
```

Comment:
Add another comment here:
`/** Evaluates convective flux based on the chosen split form.`